product_docs/docs/pgd/5/durability/lag-control.mdx

```diff
@@ -11,7 +11,7 @@ time or chronically in high availability applications, then
 organizational objectives related to disaster recovery or business
 continuity plans might not be satisfied.
 
-The replication lag control (RLC) feature is designed to regulate this
+The replication Lag Control (RLC) feature is designed to regulate this
```

Comment:
Should "Replication" also start with an uppercase letter in this case, i.e. when we abbreviate it to RLC? It seems later on, we don't ever use RLC, but only use "Lag Control" (w/o the R word).

Reply:
I think just calling it Lag Control might cover it and then we don't need the acronym.